jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PropertyNames.java

```diff
@@ -42,6 +42,8 @@
   public static final String CONTAINER_PORTS = "jib.container.ports";
   public static final String CONTAINER_USE_CURRENT_TIMESTAMP = "jib.container.useCurrentTimestamp";
   public static final String USE_ONLY_PROJECT_CACHE = "jib.useOnlyProjectCache";
+  public static final String BASE_IMAGE_LAYERS_CACHE = "jib.baseImageCache";
+  public static final String APPLICATION_IMAGE_LAYERS_CACHE = "jib.applicationImageCache";
```

Comment:
I'm being nitty, but the application is not an "image" right? Just curious if `applicationCache` makes sense?

Reply:
Isn't the concept of layering supposed to be hidden from plugin users though?

Reply:
Yea, either way would be fine.

Reply:
Since this is in plugins common, I think I'll keep it this way then (`jib.baseImageCache` and `jib.applicationCache`). Plus it's less verbose which is nice.